docs/docs/migration.md

```diff
@@ -51,12 +51,53 @@ You need to update your `mta.yaml` file to exclude `html5` modules from the resu
         properties:
           url: ${default-url}
 ```
-   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; After renaming, make sure that the   places where the name is used refer to the correct artifact.
-
-   
+After renaming, make sure that the places where the name is used refer to the correct artifact.
   
 ---
 **NOTE**
 If you try to build the project without these changes, the build tool will identify this and will fail the build with the corresponding errors.
 
 ---
+
+* `JSON` files with [service creation parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) or [service binding parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) referenced by `path` property of the correponding entity in the `mta.yaml` are packaged differently into the result MTA archive. 
+
+Therefore, if your `JSON` file contains [parameters or placeholders](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/490c8f71e2b74bc0a59302cada66117c.html) that should be resolved when you deploy the MTA archive, the correponding properties should be moved to the `mta.yaml` file. Otherwise, values assigned to these properties during deploymnet will be incorrect, since the [parameters or placeholders](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/490c8f71e2b74bc0a59302cada66117c.html) are resolved only if they are specified within an MTA descriptor, i.e. the `mta.yaml` or `mtad.yaml` files.  &nbsp;
```

Comment:
I think the spacing makes it so this (and the rest of the page) is outside the bullet point that starts with `JSON`

Reply:
Fixed